addons/web/static/src/views/pivot/pivot_view.js

```diff
@@ -129,8 +129,9 @@ export class PivotView extends Component {
         });
 
         // retrieve form and list view ids from the action
+        const views = this.env.actionContext.views || [];
```

Comment:
I'd prefer actionContext not to be a requirement in the env (for instance, for hypothetical standalone views in the frontend)

Reply:
or should we enforce it?